### PR TITLE
Creating a stub of the file for London

### PR DESCRIPTION
**Pull request by iamdv**
This file contains information about great places to eat in London and also fun things to do

### CODE REVIEW

**Comment by loupendley** [comment]
Looks good :+1:

**Comment by iamdv** [comment]
Cheers

File: places/london.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## London, UK
+
+### Great Place to Eat
+
+- Vapiano near London Bridge is a great Italian place with comfortable seating!
```
Comment (iamdv) [inline places/london.md:5]: @lp110118 updated Italian restaurant.

**Comment by patray** [comment]
Looks good.

**Comment by iamdv** [comment]
cheers @patray

**Pull request merged**